### PR TITLE
[office-js-preview] Pulling latest type definitions from CDN

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: Beta URL: https://appsforoffice.microsoft.com/lib/beta/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -71436,6 +71512,14 @@ declare namespace Excel {
...
              */
             rotation?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Specifies the share link to an Office Script file on OneDrive that will be associated with this shape.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            scriptLink?: boolean;
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:71522]: @wandyezj, Why is Script stuff showing up in the public d.ts?

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
@alison-mk, All the content for WorkbookProtection looks great. Can you delve into the `scriptLink` additions? They seem like an internal thing, but maybe I'm mistaken.

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (wandyezj) [reply types/office-js-preview/index.d.ts:71522]: This is an Add-In API. It should be ok for this API to be in beta.

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -71628,6 +71712,14 @@ declare namespace Excel {
...
              */
             rotation?: boolean;
+            /**
+             *
+             * Specifies the share link to an Office Script file on OneDrive that will be associated with this shape.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            scriptLink?: boolean;
```
Comment (wandyezj) [inline types/office-js-preview/index.d.ts:71722]: scriptLink should now show up.

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (alison-mk) [reply types/office-js-preview/index.d.ts]: Removed, resolving conversation.

File: types/office-js-preview/index.d.ts (hunk 2 above)
Comment (alison-mk) [reply types/office-js-preview/index.d.ts]: Removed scriptLink

**Comment by alison-mk** [comment]
@AlexJerabek I removed scriptLink per discussion with Joe, can you take another look at this PR?

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**